lib/gooddata/lcm/actions/create_segment_masters.rb

```diff
@@ -82,7 +109,7 @@ def call(params)
               ds.segment_id == segment_id
             end
 
-            # Create new master project
+              # Create new master project
```

Comment:
warning: Layout/CommentIndentation: Incorrect indentation detected (column 14 instead of 12). (rubocop)